src/components/Avatar/Avatar.stories.tsx

```diff
@@ -91,6 +177,12 @@ Avatar_Default.args = {
   alt: imageProps.alt,
 };
 
+Avatar_StatusIcon.args = {
+  ...avatarArgs,
+  type: 'round',
+  size: '100px',
```

Comment:
Not sure it is possible because `Icon` (`default size=IconSize.Medium=20px`) and `Avatar` (`default size=32px`) already have default values, which, without modification do not conform to the design sizes (`16px` and `100px` respectively).

Reply:
Added a function which calculates the "status item size and padding" based on "Avatar size".